File: CS/QB/QBForm.cs
using System;
using System.Windows.Forms;
using ConsoleProviderCustomization;
using DevExpress.DataAccess.ConnectionParameters;
using DevExpress.DataAccess.Native.Sql;
using DevExpress.DataAccess.Native.Sql.ConnectionProviders;
using DevExpress.DataAccess.Sql;
using DevExpress.DataAccess.UI.Sql;

namespace QB
{
    public partial class QBForm : Form
    {
        public QBForm()
        {
            InitializeComponent();
        }

        void buttonsRunQb_Click(object sender, EventArgs e)
        {
            DataAccessMSSqlConnectionProvider.ProviderRegister();
            CustomMsSqlConnectionProvider.RegProvider();
            ColumnarDataLoaderFactory.RegisterCustomColumnarDataLoader<CustomMsSqlConnectionProvider>(provider => new CustomColumnDataLoader(provider));
            MsSqlConnectionParameters connectionParameters = new MsSqlConnectionParameters("server_name", "database_name", "username", "password", MsSqlAuthorizationType.SqlServer);
            SqlDataSource yourDataSource = new SqlDataSource(connectionParameters);
            yourDataSource.AddQuery();
        }
    }
}

File: VB/QB/QBForm.vb
Imports System
Imports System.Windows.Forms
Imports ConsoleProviderCustomization
Imports DevExpress.DataAccess.ConnectionParameters
Imports DevExpress.DataAccess.Native.Sql
Imports DevExpress.DataAccess.Native.Sql.ConnectionProviders
Imports DevExpress.DataAccess.Sql
Imports DevExpress.DataAccess.UI.Sql

Namespace QB
    Partial Public Class QBForm
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub buttonsRunQb_Click(ByVal sender As Object, ByVal e As EventArgs) Handles buttonsRunQb.Click
            DataAccessMSSqlConnectionProvider.ProviderRegister()
            CustomMsSqlConnectionProvider.RegProvider()
            ColumnarDataLoaderFactory.RegisterCustomColumnarDataLoader(Of CustomMsSqlConnectionProvider)(Function(provider) New CustomColumnDataLoader(provider))
            Dim connectionParameters As New MsSqlConnectionParameters("server_name", "database_name", "username", "password", MsSqlAuthorizationType.SqlServer)
            Dim yourDataSource As New SqlDataSource(connectionParameters)
            yourDataSource.AddQuery()
        End Sub
    End Class
End Namespace
